src/index.js

```diff
@@ -24,10 +24,19 @@ import { Schedule } from './components/pages/Schedule/Schedule';
 import { ManageResources } from './components/pages/ManageResources/ManageResources';
 import { MenteesProgress } from './components/pages/MenteesProgress/MenteesProgress';
 
+import { createStore, applyMiddleware } from 'redux';
+import { Provider } from 'react-redux';
+import rootReducer from './state/reducers';
+import promiseMiddleware from 'redux-promise';
```

Comment:
Interesting - I'm used to seeing `redux-thunk` brought in at this stage, and actually had to go do some reading to learn more about `redux-promise`! Well done, and thanks for keeping me on my toes! 😁